tests/pytest_tests/unit_tests/test_system_metrics/test_gpu_apple.py

```diff
@@ -49,7 +49,7 @@ def test_gpu_apple(test_settings):
         )
         assert gpu.is_available()
         gpu.start()
-        assert gpu.probe() == {"gpuapple": {"type": "arm", "vendor": "Apple"}}
+        assert gpu.probe()["gpuapple"]["vendor"] == "sppci_vendor_Apple"
```

Comment:
how did this succeed before in CI?

Reply:
we haven't been running this in ci...

Reply:
lol https://app.circleci.com/pipelines/github/wandb/wandb/33252/workflows/c2eb541e-d471-4e2b-a8ec-226292f2b6b8/jobs/1128542?invite=true#step-105-3359